dev/com.ibm.ws.security.social_fat.commonTest.LibertyOP/publish/files/serversettings/LibertyOP_minimalConfig_oidc_SocialDiscoveryConfig.xml

```diff
@@ -1,15 +1,12 @@
 <!-- 
 /*******************************************************************************
- * Copyright (c) 2021 IBM Corporation and others.
+ * Copyright (c) 2021, 2023 IBM Corporation and others.
```

Comment:
Looks like I ended up undoing any changes I had done in here, so could have left this one out.